### ISSUE TITLE
[Feature Request] Hot option "Send audio/video to device"

### ISSUE THREAD

**Issue by varlesh** [zaps166/QMPlay2#113]
Hi:) 
Yes, i know on preferencies available option for choose device, decoder and etc... but will be good if add hot option (right click menu) for toggle device (similar option available on SMPlayer).
For example, i'm plug HDMI cable, open video file on QMPlay2 and send audo or video to HDMI on my TV. It's very comphortable.
![2017-07-23 09-05-06](https://user-images.githubusercontent.com/8083855/28496843-c97445d4-6f86-11e7-9e2d-9de263b02091.png)

**Comment by zaps166** [comment]
Nice feature, for audio we need a shortcut to menu only, but for video I must change GUI - currently full screen video can be displayed only in main window, so I can do "Full screen on screen ...." w/o many changes.

**Comment by notramo** [comment]
I think it should have some options to control what to send along with the video.
For example if somebody doing a presentation with a projector, OSD should appear on the laptop screen (main window), but if somebody is watching a video on the TV, it is comfortable to see the OSD on the same screen.

**Comment by leirbag28** [comment]
Yeah!  Lets Do it!  QMPlay2 is Quickly becoming my ONLY go to Video Player.... Super shocked its the Only one that can Loop Video smoothly (using FFmpeg/FFpay correct). Later inspired me to use FFPlay as a command and creat Launchers to drag and drop videos onto to loop...I did this with QMPlay2 first. VLC cant even do that. Anyhow Yeah this feature would just ad to its awesomeness.
